assets/models/Map.js

```diff
@@ -1,53 +1,70 @@
 class Map {
-	static CURRENT_ZOOM = 11
-	static MAX_ZOOM = 19
-	static MAP_ID = 'map'
-	static DEFAULT_LOCATION = {
-		latitude: 49.2,
-		longitude: -123.12,
-	}
+  static map;
```

Comment:
map object should not be static, this was changed in a previous PR, please read the description of this PR: https://github.com/whamester/WMDD-4885-Project/pull/8, and we can talk about why there needs to be a change of instance properties and methods to static.